crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,12 +1276,20 @@ def poweron
 
   def powercycle
     set_state("reboot")
-    bmc_cmd("power cycle")
+    if @node[:platform] == "windows"
+      net_rpc_cmd(:power_cycle)
```

Comment:
From net documentation:
-f
Force shutting down all applications.

It can be executed also without -f, but to force dangling applications to close is safer.
in both  cases power cycle and reboot, poweroff shutdown respectively, the operation is the same, that's why it's submitted like this.